precheck.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+sbt clean coverage test coverageReport
```

Comment:
Just noticed that this doesn't include the integration tests (which are not set up in build.sbt).  I wonder if they work?

Reply:
I think that all tests in this service are run with `test`

@RashmithReddy Is there any other comment to run acceptance tests?

Reply:
Looks like you are right.   Just ran `sbt test` locally and it does run tests under integration folder